crypto/store/loader_file.c

```diff
@@ -10,6 +10,7 @@
 #include <string.h>
 #include <sys/stat.h>
 #include <assert.h>
+#include <ctype.h>
```

Comment:
There is tolower call in _WIN32 fragment.... Originally ossl_tolower was part of this request, but was removed. [I suppose as follow-up to suggestion to simplify it even further.] As for purpose, it wasn't actually proclaimed that the removal of references to libc ctype is the ultimate goal. It was only said that removal of dependency is justified in some, probably most, but not necessarily all cases, even in libcrypto. If you suggest explicitly to remove *all* references, then I suppose it can be discussed...